### PR TITLE
Set up a basic workflow

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: .github/workflows/extract.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+name: Check Extract Code
```
Comment (Leon-JS02) [inline .github/workflows/extract.yml:1]: I like how you've separated the different workflows into separate yml files.

File: .github/workflows/extract.yml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    # Install Python
+    - name: Install Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.12'
+
+    # Install required packages
+    - name: Install packages
+      run: |
+        pip install -r ./web_scraping/steam_scraper/requirements.txt
```
Comment (Leon-JS02) [inline .github/workflows/extract.yml:33]: I like the neatly separated requirements

**Review by MichaelHassan — APPROVED** [review]
- placeholder tests make sense but would a better name be more helpful, perhaps check readme

File: dashboard/test_dashboard.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# pylint: skip-file
+
+from os import path
```
Comment (Leon-JS02) [inline dashboard/test_dashboard.py:3]: Smart idea with the placeholder test. This can be removed when actual code is written.

**Review by Leon-JS02 — APPROVED** [review]
Good workflows.

**Pull request merged**